pkgs/applications/window-managers/dwl/default.nix

```diff
@@ -39,18 +39,18 @@ stdenv.mkDerivation rec {
     pixman
     wayland
     wayland-protocols
-    wlroots
+    wlroots_0_16
   ] ++ lib.optionals enable-xwayland [
     libX11
-    xwayland
+    xorg.xcbutilwm
   ];
 
   # Allow users to set their own list of patches
   patches = totalPatches;
 
-  # Last line of config.mk enables XWayland
```

Comment:
This won't be fun if the file gets bigger and now `13-14` is in the middle. Can we:
```
cat <<EOF >config.mk
XWAYLAND = -DXWAYLAND
XLIBS = xcb xcb-icccm
EOF
```
Or something?